ticket/models.py

```diff
@@ -47,10 +50,15 @@ class Order(models.Model):
         TicketType, related_name="orders", on_delete=models.CASCADE
     )
     quantity = models.PositiveIntegerField()
-    fulfilled = models.BooleanField(default=False)
+    state = models.CharField(
+        max_length=20,
+        choices=constants.ORDER_STATES,
+        default=constants.CREATED_STATE,
+    )
+    state_change_at = models.DateTimeField(null=True)
```

Comment:
This is `null=True` only the first time the migration runs to create the fields. In later iterations `auto_add_now` should be added. Combined with the default CHOICE for state, this will gives the initial state of an Order as CREATED and when that happened. Every next change of `state` also updates `state_change_at`.

Not adding the default during creation helps run the migration faster because a default value will not be generated.